src/NServiceBus.AcceptanceTests/Sagas/When_timeout_hit_not_found_saga.cs

```diff
@@ -83,7 +83,7 @@ public void Handle(object message)
                     {
                         Context.NotFoundHandlerCalledForTimeout = true;
                     }
-
+                    return Task.FromResult(0);
```

Comment:
nope. plan is to update acceptance tests to 4.6. then no need to make taskex public